src/test/java/org/threeten/extra/AbstractDateTimeTest.java

```diff
@@ -157,25 +142,15 @@ public void basicTest_get_TemporalField_supported() {
     public void basicTest_get_TemporalField_unsupported() {
```

Comment:
It occurs to me that these can be changed to parameterized tests, if desired.  It would require changing the test lifecycle to per-class instead of per-method, since the `samples()` & `invalidFields()` methods are non-static:

```java
@TestInstance(TestInstance.Lifecycle.PER_CLASS)
public abstract class AbstractDateTimeTest {
    // ...

    protected Stream<Arguments> invalidFieldArguments() {
        return samples().stream().flatMap(sample ->
                invalidFields().stream().map(field ->
                        Arguments.of(sample, field)));
    }

    @MethodSource("invalidFieldArguments")
    @ParameterizedTest
    public void basicTest_get_TemporalField_unsupported(TemporalAccessor sample, TemporalField field) {
        assertThrows(DateTimeException.class, () -> sample.get(field), "Failed on " + sample + " " + field);
    }

    @MethodSource("samples")
    @ParameterizedTest
    public void basicTest_get_TemporalField_null(TemporalAccessor sample) {
        assertThrows(NullPointerException.class, () -> sample.get(null), "Failed on " + sample);
    }

    // ...
}
```